Author: aszlig <aszlig@nix.build>
Date:   Wed May 2 03:15:04 2018 +0200

    Use substvar in migration runner
    
    This is so that we can later replace this using stdenv's substitute.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: migration-substvar.patch

diff --git a/migrations/migration-runner.js b/migrations/migration-runner.js
index bce68392e..92c5855b8 100644
--- a/migrations/migration-runner.js
+++ b/migrations/migration-runner.js
@@ -17,5 +17,5 @@ function setUpServer () {
 setUpServer();
 
 // Replace this with your migration
-const processUsers = require('./groups/reconcile-group-plan-members.js');
+const processUsers = require('@migrationScript@');
 processUsers();
